components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -1535,6 +1536,9 @@ private void addServiceProviderOIDCProperties(Connection connection,
             addToBatchForOIDCPropertyAdd(processedClientId, spTenantId, prepStmtAddOIDCProperty,
                     SUBJECT_TYPE, consumerAppDO.getSubjectType());
 
+            addToBatchForOIDCPropertyAdd(processedClientId, spTenantId, prepStmtAddOIDCProperty,
+                    IS_FAPI_CONFORMANT_APP, consumerAppDO.getSubjectType());
```

Comment:
`consumerAppDO.getSubjectType()` is this correct?

Reply:
fixed